lua/entities/gmod_wire_expression2/core/custom/tardis2.lua

```diff
@@ -16,11 +16,15 @@ local function getTardis(ent)
     end
 end
 
-local function HandleE2(ent, name, ...)
+local function HandleE2(ent, type, name, e2, ...)
     if IsValid(getTardis(ent)) then
-        return ent:HandleE2(name, ...)
+        if type == "Setter" and ent:CheckSecurity(e2.player) == false then
+            TARDIS:ErrorMessage(e2.player,"Security.E2.UseDenied")
```

Comment:
How does this behave in E2 if it's expected to get something back but returns nothing now - does it cause any issues?

Reply:
From what I tested, the E2 just does nothing